rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/message/autocomplete/user/UserViewHolder.java

```diff
@@ -16,6 +20,12 @@
   private final RocketChatAvatar avatar;
   private final ImageView status;
 
+  private static final int[] COLORS = new int[] {
```

Comment:
Yes, for the same reason [bellow](https://github.com/RocketChat/Rocket.Chat.Android/pull/372#discussion_r131030142).

:(